docker/deployable-zipfile/Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN mkdir -p ${HOME} && chmod 777 ${HOME}
 # Python 3 to be enabled at login.
 RUN yum -y update && \
     yum install -y centos-release-scl && \
+    rpm -i https://rpm.nodesource.com/pub_14.x/el/7/x86_64/nodejs-14.18.1-1nodesource.x86_64.rpm && \
```

Comment:
I think I'd prefer leaving out this change (letting it get reviewed/merged in a separate PR), since it's unrelated.